com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleVirtualTextureNode.cs

```diff
@@ -272,12 +272,13 @@ public SampleVirtualTextureNodeSettingsView(SampleVirtualTextureNode node)
                 });
 
                 // display warning if the current master node doesn't support virtual texturing
-                if (!m_Node.owner.isSubGraph)
-                {
-                    bool supportedByMasterNode = m_Node.owner.GetNodes<IMasterNode>().FirstOrDefault()?.supportsVirtualTexturing ?? false;
-                    if (!supportedByMasterNode)
-                        ps.Add(new HelpBoxRow(MessageType.Warning), (row) => row.Add(new Label("The current master node does not support Virtual Texturing, this node will do regular 2D sampling.")));
-                }
+                //TODO: Fix the fact bool passing that happened with master nodes to now happen with targets/subtargets
```

Comment:
So with no idea of master nodes now, need to probably put this bool flag on either targets or subtargets